mods/ra/rules/aircraft.yaml

```diff
@@ -434,3 +434,96 @@ U2:
 	RejectsOrders:
 	Interactable:
 	-MapEditorData:
+
+BLCK:
```

Comment:
Please either call this `mh60` or rename the sequence and sprite to `blck`. There is no need for these to be inconsistent.

Reply:
for now I named it to mh60